pos_default_empty_image/README.rst

```diff
@@ -0,0 +1,95 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+POS Default Empty Image
+=======================
+
+This module extends the functionality of point of sale to make PoS load faster
+and to improve products display.
+
+Point Of Sale Load faster
+-------------------------
+
+In the point of sale, trying to load known inexistant images is a waste of time.
```

Comment:
rephrase to remove "waste of time" (eg: "loading product without image spend unnecessary time and resources.")